podcasts/Onboarding/Plus/UpgradeCard.swift

```diff
@@ -88,9 +88,10 @@ struct UpgradeCard: View {
                                 .aspectRatio(contentMode: .fit)
                                 .foregroundColor(theme.primaryText01)
                                 .frame(width: 16, height: 16)
-                            Text(feature.title)
+                            UnderlineLinkTextView(feature.title)
                                 .font(size: 14, style: .subheadline, weight: .medium)
                                 .foregroundColor(theme.primaryText01)
+                                .tint(.black)
```

Comment:
You should use a theme color in order to work correctly on all themes. This is relevant if you activate the `newAccountUpgradeFlow` feature flag and the card is used on the account view.
![Simulator Screenshot - iPhone 15 Pro - 2024-02-22 at 16 58 55](https://github.com/Automattic/pocket-casts-ios/assets/651601/1e53e223-df94-439a-a3dd-f75d99252248)

Reply:
Done in f6775f02d0e6dd2fb8c2aa6249398029dd0bf490